### DISCUSSION TITLE
How do I view the built site?

### DISCUSSION THREAD

**Discussion by willhiorns** [UKHomeOffice-test/docs-as-code-nunjucks#30] [General]
It's not clear to me how to get to the site.  I clicked on the link on repo home page (https://ukhomeoffice-test.github.io/docs-as-code-nunjucks/) and I go to a single page with some content ... but no links ... and where are the Principles?

The "Environments" page makes me think that it _has_ all been built and published as a Page: https://github.com/UKHomeOffice-test/docs-as-code-nunjucks/deployments/activity_log?environment=github-pages

Help?

n00b

**Comment by edhamiltonHO** [comment]
Yeah, not put any effort into site structure yet. Something we very much need to do.

In meantime you can click through some basic things here https://ukhomeoffice-test.github.io/docs-as-code-nunjucks/principles/
